packages/contracts-ops/contracts/test/Reboot.t.sol

```diff
@@ -27,56 +33,70 @@ contract RebootTest is
             NomadTestWithUpdaterManager,
             ReplicaHandlers,
             ReplicaTest,
-            HomeTest
+            HomeTest,
+            GovernanceRouterTest
         )
     {
+        // ALL
         vm.createSelectFork(vm.envString("RPC_URL"), 15_977_625);
```

Comment:
This function is too large, I can't tell what it's doing. Break it up into discrete steps

Reply:
rule of thumb: anything that invokes writejson should be in a different scope that pairs it with the thing that generated whatever it is that is being written